recipes/pycirclize/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pycirclize" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycirclize-{{ version }}.tar.gz
+  sha256: 7f83e406636cee351788162a0a772b1cf91854067c67ca1942609ad141b5e1a4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - poetry
+    - pip
+  run:
+    - python >=3.8,<4.0
+    - biopython >=1.79,<2.0
+    - matplotlib-base >=3.5.2,<4.0.0
+    - numpy >=1.21.1,<2.0.0
+    - pandas >=1.3.5,<2.0.0
+
+test:
+  imports:
+    - pycirclize
+  commands:
+    - pip check
```

Just go ahead and add here:
```suggestion
    - pip check
    - pytest tests
```

and remove `run_test.sh`.

---

Thanks for the review. Fixed.